modules/apps/portal-crypto/portal-crypto-hash-api/src/main/java/com/liferay/portal/crypto/hash/request/command/pepper/PepperCommand.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.crypto.hash.request.command.pepper;
+
+/**
+ * @author Carlos Sierra Andrés
+ */
+public class PepperCommand {
```

Comment:
I am not sure we need a `PepperCommand`... do we ever expect to have several options for the pepper as we have for the salt?

Reply:
No, I just feel like we should treat every add-on equally. You know, since salt has it's wrapper, pepper should have it too, even though it's only for storing a pepper field, so that the whole thing looks more structured.
Can we keep it or we really dont need it?

Reply:
sure... I am fine with keeping it